end-to-end-tests/package.json

```diff
@@ -45,9 +49,9 @@
     "@latticexyz/world": "2.0.0-transaction-context-98ef570f",
     "@latticexyz/world-modules": "2.0.0-transaction-context-98ef570f",
     "@latticexyz/gas-report": "2.0.0-transaction-context-98ef570f",
-    "@eveworld/smart-object-framework": "0.0.1",
-    "@eveworld/world": "0.0.1",
-    "@eveworld/common-constants": "0.0.1"
+    "@eveworld/smart-object-framework": "0.0.2",
```

Comment:
Lets remove these changes since develop now has a direct link to the source packages